packages/astro/test/units/routing/manifest.test.js

```diff
@@ -79,11 +81,11 @@ describe('routing - createRouteManifest', () => {
 		settings.injectedRoutes = [
 			{
 				pattern: '/about',
-				entrypoint: '@lib/legacy/static.astro',
+				entrypoint: 'src/entrypoint.astro',
```

Comment:
Had to update all the tests here to use a real route instead of a non-existing entrypoint, as building the manifest now read the file's content to know if it's prerendered or not. Doing this in Astro actually doesn't work without hacks anyway, as we'll always try to resolve the path before Vite is started.